### PR TITLE
Scaffolding for splitting up the sidebar into documentation-sidebar and getting-started-sidebar

### PR DESCRIPTION
**Pull request by jin**

### CODE REVIEW

File: site/_config.yml [hunk 1]
```diff
@@ -64,6 +65,11 @@ defaults:
     values:
       toc: true
       sitemap: true
+  -
```
Comment (jin) [inline site/_config.yml:68]: This sets the default `category` frontmatter variable for all pages to be `documentation`.

File: site/_includes/header.html [hunk 2]
```diff
@@ -27,6 +27,9 @@
             </div>
           </form>
           <ul class="nav navbar-nav navbar-right">
+            <li{% if page.nav == "docs" %} class="active"{% endif %}>
```
Comment (jin) [inline site/_includes/header.html:30]: This adds a top level navbar item to the `site.getting_started_category_url` variable set in `_config.yml`.

File: site/_layouts/documentation.html [hunk 3]
```diff
@@ -43,10 +43,10 @@
 
             <!-- If the current version has a explicitly versioned sidebar for the major version -->
             {% if site.doc_versions contains current_version %}
-                {% capture sidebar %}documentation-sidebar/{{major_version}}/sidebar.html{% endcapture %}
+                {% capture sidebar %}{{page.category}}-sidebar/{{major_version}}/sidebar.html{% endcapture %}
```
Comment (jin) [inline site/_layouts/documentation.html:46]: This dynamically loads a different sidebar based on a page's category.

File: site/docs/guide.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: documentation
 title: User's guide
+category: getting-started
```
Comment (jin) [inline site/docs/guide.md:4]: This sets the page's category.

File: site/_includes/getting-started-sidebar/master/sidebar.html [hunk 5]
```diff
@@ -0,0 +1 @@
+This is the getting started sidebar.
```
Comment (jin) [inline site/_includes/getting-started-sidebar/master/sidebar.html:1]: This is the sidebar include partial for pages in the `getting-started` category.

File: site/_layouts/documentation.html (hunk 3 above)
Comment (jin) [reply site/_layouts/documentation.html:46]: So if a page doesn't set `getting-started` as a category, it defaults to `documentation`, which will load `documentation-sidebar/<version>/sidebar.html`.

**Review by floriographygoth — APPROVED** [review]

**Comment by aiuto** [comment]
Is this active by itself or is it being done as part of the larger docs overhaul.

**Comment by jin** [comment]
This was for assisting @floriographygoth with the nav bar overhaul, which is live on the site now.

**Pull request closed without merging**